example/src/index.html.js

```diff
@@ -40,13 +42,19 @@ function getMessage() {
   }
 }
 
-export default main
+export default withPolyfill(main)
 
-function main ({ location, data }) {
+function main ({ location, data, polyfill }) {
   if (!data) return null
   return (
     <html lang='en'>
-      { head('Rendered on server') }
```

Comment:
Instead of replacing this call with it's content, you could pass the `polyfill` function in